CRM/Contribute/Page/ContributionPage.php

```diff
@@ -182,6 +182,11 @@ public function &configureActionLinks() {
           'uniqueName' => 'pcp',
         ),
       );
+      $context = array(
+        'urlString' => $urlString,
+        'urlParams' => $urlParams,
+      );
+      CRM_Utils_Hook::tabset('civicrm/admin/contribute', self::$_configureActionLinks, $context);
```

Comment:
I'm unclear about this call - what is it doing? The first argument is the same as the one called from ContributionPage/TabHeader.php but the `$context` is different - this may lead to some confusion by hook implementers.